doc/Manuals/ug-ss_x-road_6_security_server_user_guide.md

```diff
@@ -1819,23 +1845,83 @@ It is expected that the restore command is run by the xroad user.
 In order to restore configuration, the following command should be used:
 
     /usr/share/xroad/scripts/restore_xroad_proxy_configuration.sh \
-    -s <security server ID> -f <path + filename>
+    -s <security server ID> -f <path + filename> [-P -N]
 
 For example (all on one line):
 
     /usr/share/xroad/scripts/restore_xroad_proxy_configuration.sh \
     -s AA/GOV/TS1OWNER/TS1 \
-    –f /var/lib/xroad/backup/conf_backup_20140703-110438.tar
+    –f /var/lib/xroad/backup/conf_backup_20140703-110438.gpg
+
+In case original backup encryption and siging key is lost additional parameters can be specified to skip decryption and/or
+signature verification. Use `-P` command line switch when backup archive is already decrypted externally and `-N` switch to
+skip checking archive signature.
 
-If it is absolutely necessary to restore the system from a backup made on a different security server, the forced mode of the restore command can be used with the –F option. For example (all on one line):
+If it is absolutely necessary to restore the system from a backup made on a different security server, the forced mode
+of the restore command can be used with the –F option together with unencrypted backup archive flags. For example (all on one line):
 
     /usr/share/xroad/scripts/restore_xroad_proxy_configuration.sh \
-    -F –f /var/lib/xroad/backup/conf_backup_20140703-110438.tar
+    -F -P –f /var/lib/xroad/backup/conf_backup_20140703-110438.tar
+
+In case backup archives were encrypted they have to be first unencrypted in external safe environment and then securely
+transported to security server filesystem.
 
 ### 13.3 Automatic Backups
 
-By default the Security Server backs up its configuration automatically once every day. Backups older than 30 days are automatically removed from the server. If needed, the automatic backup policies can be adjusted by editing the `/etc/cron.d/xroad-proxy` file.
+By default the Security Server backs up its configuration automatically once every day. Backups older than 30 days are
+automatically removed from the server. If needed, the automatic backup policies can be adjusted by editing the
+`/etc/cron.d/xroad-proxy` file.
+
+### 13.4 Backup Encryption Configuration
+
+Back up encryption is initially turned off. To turn encryption on configuration must be overridden in the file `/etc/xroad/conf.d/local.ini`, in the `[proxy]` section.
+
+    [proxy]
+    backup-encrypted=false
+    backup-public-key-path=/etc/xroad/backupkeys
+
+Backups are always signed. To turn on backup encryption add or change the `backup-encrypted` key to true. If needed also 
+change the folder where additional backup encryption keys are stored. Default is `/etc/xroad/backupkeys`.
```

Comment:
I would change this part like this:

To turn backup encryption on, please change the `backup-encrypted` property value to `true`. By default, additional encryption keys are stored in the `/etc/xroad/backupkeys` directory. The default directory can be changed by modifying the `backup-public-key-path` property value.